### PR TITLE
Query: integration test

### PR DESCRIPTION
**Pull request by smacker**
Fixes: #93 

Use gemini `./hash` for meta data.
Loads hashtables from a dump.
Checks query returns results for both identical & similar files.

Unit test was updated to reuse the same data.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,13 +57,29 @@ matrix:
       - MASTER=spark://127.0.0.1:7077
     install:
       - make build
+      - 
     script:
+      # start dependencies
       - ./scripts/get_apache_spark.sh "2.2.0" "2.7" || travis_terminate 1
       - ./scripts/start_apache_spark_cluster.sh "127.0.0.1" "7077" || travis_terminate 1
+      # python will be installed after https://github.com/src-d/gemini/issues/60
+      # and after merge https://github.com/src-d/gemini/pull/98 we can use local fe runner
```
Comment (bzz) [inline .travis.yml]: If this PR gets merged first - shall TODOs be added to #60 and #98 to take care of this?

File: src/test/resources/consumer.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
+package borges
```
Comment (bzz) [inline src/test/resources/consumer.go]: As discussed in https://github.com/src-d/gemini/pull/97#issuecomment-381856684 - in my experience, it's safer to be conservative and develop a habit of adding a comment with original license and, if possible, an origin of source code files, if copied from other projects.

File: .travis.yml [hunk 3]
```diff
@@ -57,13 +57,29 @@ matrix:
...
+      - docker build -t srcd/gemini-fe -f FE.Dockerfile .
+      - ./scripts/start_docker_fe.sh
       - sleep 10 # wait for it doesn't work, port is avalable but scyllaDB is still down
       - ./hash src/test/resources/siva || travis_terminate 1
+      # query without similarity
       - ./query ./src/test/resources/LICENSE
+      # query with similarity
+      - docker cp ./src/test/resources/hashetabes_dump.cql db:/hashetabes_dump.cql
+      - docker exec -it db cqlsh -e "SOURCE '/hashetabes_dump.cql'"
+      - ./query --doc-freq-file=src/test/resources/docfreq.json --params-file=src/test/resources/params.json --hashtables-num=9 --band-size=13 ./src/test/resources/consumer.go | tee query_result.txt
```
Comment (bzz) [inline .travis.yml]: `# query with similarity`

File: .travis.yml [hunk 4]
```diff
@@ -57,13 +57,29 @@ matrix:
...
       - ./scripts/start_apache_spark_cluster.sh "127.0.0.1" "7077" || travis_terminate 1
+      # python will be installed after https://github.com/src-d/gemini/issues/60
+      # and after merge https://github.com/src-d/gemini/pull/98 we can use local fe runner
+      - docker build -t srcd/gemini-fe -f FE.Dockerfile .
+      - ./scripts/start_docker_fe.sh
       - sleep 10 # wait for it doesn't work, port is avalable but scyllaDB is still down
       - ./hash src/test/resources/siva || travis_terminate 1
+      # query without similarity
       - ./query ./src/test/resources/LICENSE
+      # query with similarity
```
Comment (bzz) [inline .travis.yml]: `#Load hashtables from apollo DB dump`

File: .travis.yml (hunk 1 above)
Comment (smacker) [reply .travis.yml]: idk which one is going to be merged first. This PR contains lines that should be replaced later. That's why the comment is here.

File: src/test/resources/consumer.go (hunk 2 above)
Comment (smacker) [reply src/test/resources/consumer.go]: than it wouldn't be identical

File: .travis.yml (hunk 4 above)
Comment (smacker) [reply .travis.yml]: it's obvious from the command that it loads dump into DB, no matter actually where the fixture comes from.
I added comments to split groups of commands by what they do, not rewrote the command in English.

**Comment by bzz** [comment]
@smacker shall we merge, after the typo in filename is fixed?

**Comment by smacker** [comment]
fixed the typo, rebased on master. Will merge after CI is green.

**Pull request merged**